From b8d8120e6ff70d7c9353dbeeac3a0fc72c83c4d9 Mon Sep 17 00:00:00 2001
From: Hemant Singal <hemants.iitk@gmail.com>
Date: Fri, 16 Dec 2011 14:55:43 +0530
Subject: [PATCH 2/2] latest

---
 settings.py |   18 +++++++++++++-----
 urls.py     |   10 +++++++---
 2 files changed, 20 insertions(+), 8 deletions(-)

diff --git a/settings.py b/settings.py
index 5e4d3cc..2ed71a2 100644
--- a/settings.py
+++ b/settings.py
@@ -11,10 +11,10 @@ MANAGERS = ADMINS
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.', # Add 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': '',                      # Or path to database file if using sqlite3.
-        'USER': '',                      # Not used with sqlite3.
-        'PASSWORD': '',                  # Not used with sqlite3.
+        'ENGINE': 'django.db.backends.mysql', # Add 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
+        'NAME': 'natourpress',           # Or path to database file if using sqlite3.
+        'USER': 'natourpress',           # Not used with sqlite3.
+        'PASSWORD': 'natourpress',       # Not used with sqlite3.
         'HOST': '',                      # Set to empty string for localhost. Not used with sqlite3.
         'PORT': '',                      # Set to empty string for default. Not used with sqlite3.
     }
@@ -29,6 +29,10 @@ DATABASES = {
 # system time zone.
 TIME_ZONE = 'America/Chicago'
 
+LOGIN_URL='/login/'
+
+LOGIN_REDIRECT_URL='/feeds/'
+
 # Language code for this installation. All choices can be found here:
 # http://www.i18nguy.com/unicode/language-identifiers.html
 LANGUAGE_CODE = 'en-us'
@@ -69,6 +73,7 @@ ADMIN_MEDIA_PREFIX = '/static/admin/'
 
 # Additional locations of static files
 STATICFILES_DIRS = (
+    '/home/environments/natourpress/src/django-natourpress/natourpress/templates',
     # Put strings here, like "/home/html/static" or "C:/www/django/static".
     # Always use forward slashes, even on Windows.
     # Don't forget to use absolute paths, not relative paths.
@@ -106,6 +111,7 @@ TEMPLATE_DIRS = (
     # Put strings here, like "/home/html/django_templates" or "C:/www/django/templates".
     # Always use forward slashes, even on Windows.
     # Don't forget to use absolute paths, not relative paths.
+    '/home/environments/natourpress/src/django-natourpress/natourpress/templates',
 )
 
 INSTALLED_APPS = (
@@ -115,6 +121,8 @@ INSTALLED_APPS = (
     'django.contrib.sites',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.admin',
+    'natourpress',
     # Uncomment the next line to enable the admin:
     # 'django.contrib.admin',
     # Uncomment the next line to enable admin documentation:
diff --git a/urls.py b/urls.py
index be62625..f8fa06a 100644
--- a/urls.py
+++ b/urls.py
@@ -1,8 +1,8 @@
 from django.conf.urls.defaults import patterns, include, url
 
 # Uncomment the next two lines to enable the admin:
-# from django.contrib import admin
-# admin.autodiscover()
+from django.contrib import admin
+admin.autodiscover()
 
 urlpatterns = patterns('',
     # Examples:
@@ -13,5 +13,9 @@ urlpatterns = patterns('',
     # url(r'^admin/doc/', include('django.contrib.admindocs.urls')),
 
     # Uncomment the next line to enable the admin:
-    # url(r'^admin/', include(admin.site.urls)),
+    url(r'^config/', include('natourpress.urls')),
+    url(r'^admin/', include(admin.site.urls)),
+   # url(r'^login', 'natourpress.views.my_login'),
+    url(r'^login', 'django.contrib.auth.views.login',{'template_name': 'natourpress/login.html'}),
+    url(r'^logout', 'natourpress.views.logout_view'),
 )
-- 
1.7.5.4

